test/boulder-config.json

```diff
@@ -119,11 +120,13 @@
 
   "ra": {
     "rateLimitPoliciesFilename": "test/rate-limit-policies.yml",
+    "maxConcurrentRPCServerRequests": 0,
```

Comment:
For boulder-ca, the results from pkcs11bench suggested that we should do 16 concurrent goroutines. I'd suggest starting with that for each server.

@rolandshoemaker: Can you set these to 16 in this config?